### PR TITLE
fix(pci.kube.oidc): update translation files

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-58923
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
Update OIDC Provider translations

## Related
- [x] TRANS-39665

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "pci_projects_project_kubernetes_details_service_add_oidc_provider_title": "Configurer le fournisseur OIDC",
-  "pci_projects_project_kubernetes_details_service_add_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OpenID Connect comme Dex, Keycloack ou un fournisseur SaaS, à la condition que soient supportés la fonctionnalité \"OpenID Connect discovery\", le TLS et un certificat CA signé.",
+  "pci_projects_project_kubernetes_details_service_add_oidc_provider_description": "Vous pouvez utilisez la plupart des fournisseurs OIDC comme Dex, Keycloak ou un fournisseur SaaS, à la condition qu'ils exposent la fonctionnalité \"OpenID Connect discovery” . Le fournisseur OIDC doit être accessible depuis une URL en HTTPS et un certificat CA signé est obligatoire.",
```
Comment (frenautvh) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/translations/Messages_fr_FR.json:3]: vous pouvez utiliser

**Review by frenautvh — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/translations/Messages_fr_FR.json (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/translations/Messages_fr_FR.json]: Thanks.

**Review by anooparveti — DISMISSED** [review]

**Review by varun257 — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "pci_projects_project_kubernetes_details_service_add_oidc_provider_title": "Configurer le fournisseur OIDC",
-  "pci_projects_project_kubernetes_details_service_add_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OpenID Connect comme Dex, Keycloack ou un fournisseur SaaS, à la condition que soient supportés la fonctionnalité \"OpenID Connect discovery\", le TLS et un certificat CA signé.",
+  "pci_projects_project_kubernetes_details_service_add_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OIDC comme Dex, Keycloak ou un fournisseur SaaS, à la condition qu'ils exposent la fonctionnalité \"OpenID Connect discovery” . Le fournisseur OIDC doit être accessible depuis une URL en HTTPS et un certificat CA signé est obligatoire.",
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/translations/Messages_fr_FR.json:3]:
```suggestion
  "pci_projects_project_kubernetes_details_service_add_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OIDC comme Dex, Keycloak ou un fournisseur SaaS, à la condition qu'ils exposent la fonctionnalité \"OpenID Connect discovery”. Le fournisseur OIDC doit être accessible depuis une URL en HTTPS et un certificat CA signé est obligatoire.",
```

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/update-oidc-provider/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "pci_projects_project_kubernetes_details_service_update_oidc_provider_title": "Modifier le fournisseur OIDC",
-  "pci_projects_project_kubernetes_details_service_update_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OpenID Connect comme Dex, Keycloack ou un fournisseur SaaS, à la condition que soient supportés la fonctionnalité \"OpenID Connect discovery\", le TLS et un certificat CA signé.",
+  "pci_projects_project_kubernetes_details_service_update_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OIDC comme Dex, Keycloak ou un fournisseur SaaS, à la condition qu'ils exposent la fonctionnalité \"OpenID Connect discovery” . Le fournisseur OIDC doit être accessible depuis une URL en HTTPS et un certificat CA signé est obligatoire.",
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/update-oidc-provider/translations/Messages_fr_FR.json:3]: Same here
```suggestion
  "pci_projects_project_kubernetes_details_service_update_oidc_provider_description": "Vous pouvez utiliser la plupart des fournisseurs OIDC comme Dex, Keycloak ou un fournisseur SaaS, à la condition qu'ils exposent la fonctionnalité \"OpenID Connect discovery”. Le fournisseur OIDC doit être accessible depuis une URL en HTTPS et un certificat CA signé est obligatoire.",
```

**Review by radireddy — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**